src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/mathfuncs.txt

```diff
@@ -263,58 +263,77 @@ Blank()
 #Error
 
 >>IsNumeric("A")
+/*SqlRunner*/ Compile Error
```

Comment:
Is this jsut a case of nyi?  We need to factor this file better for this. (in the future, not blocking this PR)

Reply:
This is relatively new, and  not implemented in SQL yet